src/Microsoft.AspNet.Mvc.Core/ActionResults/PartialViewResult.cs

```diff
@@ -50,10 +51,22 @@ public class PartialViewResult : ActionResult
             var viewEngine = ViewEngine ??
                              context.HttpContext.RequestServices.GetRequiredService<ICompositeViewEngine>();
 
+            var logger = context.HttpContext.RequestServices.GetRequiredService<ILoggerFactory>()
+                .CreateLogger<PartialViewResult>();
+
             var viewName = ViewName ?? context.ActionDescriptor.Name;
-            var view = viewEngine.FindPartialView(context, viewName)
-                                 .EnsureSuccessful()
-                                 .View;
+            var viewEngineResult = viewEngine.FindPartialView(context, viewName);
+            if (!viewEngineResult.Success)
+            {
+                logger.LogError(
+                    "The partial view '{PartialViewName}' was not found. Searched locations: {SearchedViewLocations}",
+                    viewName,
+                    viewEngineResult.SearchedLocations);
+            }
+
+            var view = viewEngineResult.EnsureSuccessful().View;
+
+            logger.LogVerbose("The partial view '{PartialViewName}' was found successfully.", viewName);
```

Comment:
I initially started with logging at view engine, but avoided it because 
- since its at a very low level, i couldn't find a way to provide enough information(ex: in what context is a partial view being executed?..like a partial view can be executed from Html.Partial, PartialViewResult or a ViewComponent too).
- The probing that *we* do for Display/Edit templates is causing lot of log entries to be created which is unnecessary.

We(@rynowak, @pranavkm and myself) discussed about this and felt that logging top level things make sense..ex: ViewResult, PartialViewResult...we can avoid logging for Html.Partial and ViewComponent's views...thoughts?